packages/shared/src/components/fields/PasswordField.tsx

```diff
@@ -39,14 +42,18 @@ export function PasswordField({
   className,
   ...props
 }: PasswordFieldProps): ReactElement {
+  const onboardingPapercuts = useFeature(featureOnboardingPapercuts);
+  const errorMessage = onboardingPapercuts
+    ? 'Password needs to have at lease one uppercase, one special character and one number'
```

Comment:
Is this actually the case though XD?

Reply:
Hopefully 🤣

Reply:
I mean the password actually doesn't need to contain special char/number no?
AFAIK

Reply:
Ah, right. That is actually not the full criteria. That is recommended, but not forced.

Reply:
Reverted!